src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -582,11 +582,13 @@ trait_ref ::= path_generic_args_without_colons
 // Impl
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-impl_item ::= UNSAFE? IMPL generic_params type where_clause? '{' inner_attr* impl_member* '}'
-            | UNSAFE? IMPL generic_params '!'? trait_ref FOR (type where_clause? '{' inner_attr* impl_member* '}' | '..' '{' '}') {
-  pin(".*") = "'{'"
+impl_item ::= UNSAFE? IMPL generic_params type where_clause? impl_members_block
+            | UNSAFE? IMPL generic_params '!'? trait_ref FOR (type where_clause? impl_members_block | '..' '{' '}') {
+  pin(".*") = '{'
 }
 
+impl_members_block ::= '{' inner_attr* impl_member* '}' {pin=1}
```

Comment:
Well, it may be a wrong decision, because everybody else don't uses spaces here. But it is definitely not a good idea to change code style during functional changes.